packages/design/src/Typography.stories.mdx

```diff
@@ -139,7 +139,26 @@ export function PixelSize({ of: size }) {
 | `--typography--fontSize-jumbo`       | <Example of="jumbo" />       | <PixelSize of="jumbo" />       |
 | `--typography--fontSize-extravagant` | <Example of="extravagant" /> | <PixelSize of="extravagant" /> |
 
-## Line-height
+## Font-size (mobile)
+
+Jobber's mobile platform uses the following sizing tokens for typography.
+
+The mobile platform has a base font-size equivalent to `16px`, with options for
+a larger or smaller font-size depending on the use case.
+
+| Name                                 | Visual                                                                  | Value |
+| :----------------------------------- | :---------------------------------------------------------------------- | :---- |
+| `--typography--fontSize-smallest`    | <div style={{fontSize: "10px", fontFamily: "Inter"}}>smallest </div>    | 10px  |
```

Comment:
i believe if you try to grab the foundation.js you can make use of them here

Reply:
foundation.js is already being pulled for storybook - but that file isn't the one that includes mobile typography tokens.  I would need to try and also pull in foundation.native 🤔

Reply:
Ah I figured it out - silly! Fixing this now

Reply:
Okay so I can get the fontSizes nicely but getting the values isn't working.  That I'm not so concerned about considering the only changes we are planning to the px sizes are to actually match mobile to web.

lineHeight visuals also get reaaaal wonky - like the lineHeights get very very tall